lua/telescope/sorters.lua

```diff
@@ -32,12 +32,17 @@ Sorter.__index = Sorter
 ---
 --- Lower number is better (because it's like a closer match)
 --- But, any number below 0 means you want that line filtered out.
---- @field scoring_function function Function that has the interface:
---      (sorter, prompt, line): number
+---@field scoring_function function: Function that has the interface: (sorter, prompt, line): number
+---@field tags table: I actually don't know :)
```

Comment:
We can maybe chime this in for tidiness :)
```suggestion
---@field tags table: Unique tags collected at filtering for tag completion
```
`tags` was introduced to enable #626